cumulusci/tasks/datadictionary.py

```diff
@@ -348,10 +432,12 @@ def _process_object_element(self, sobject_name, element, version):
         ):
             self.sobjects[sobject_name].append(
                 SObjectDetail(
-                    version,
-                    sobject_name,
-                    element.label.text,
-                    description_elem.text if description_elem is not None else "",
+                    version=version,
+                    api_name=sobject_name,
+                    label=element.label.text,
+                    description=description_elem.text
```

Comment:
I like the use of kwargs here (and elsewhere). Definitely an improvement to readability.